commands/cdn/distribution/routingrules/routing_rules.go

```diff
@@ -0,0 +1,90 @@
+package routingrules
+
+import (
+	"context"
+	"fmt"
+	"github.com/ionos-cloud/ionosctl/v6/commands/cdn/completer"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/json2table"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/json2table/jsonpaths"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/jsontabwriter"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/tabheaders"
+	cdn "github.com/ionos-cloud/sdk-go-cdn"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	allRoutingRulesColumns     = []string{"Scheme", "Prefix", "Host", "Caching", "Waf", "RateLimitClass", "SniMode", "GeoRestrictions"}
```

Comment:
This might be just a nitpick and could be ignored...

But I would rename `GeoRestrictions` to reflect that it is a block list (whatever naming you want honestly, i.e. `GeoBlockList` or `BlacklistCountries` etc etc.)

Reasoning: I suspect the API might change in the future and they could add, for example, a whitelist

Reply:
GeoRestrictions can contain allowList or blockList. The struct contains both

Reply:
it might actually be more legible to split this column into 2, one for `allowList` and one for `blockList`, imho